backend/tracim_backend/models/context_models.py

```diff
@@ -140,7 +140,7 @@ def __init__(self, loggedin_user_password: str, email: str) -> None:
 
 
 class SimpleFile(object):
-    def __init__(self, files: cgi.FieldStorage) -> None:
+    def __init__(self, files: cgi.FieldStorage = None) -> None:
```

Comment:
Why this diff ?

Reply:
files param is not mandatory here, but is mandatory later in code, None case allow use to know there. if we do not set None by default here, code will raise 500 here instead of raising 400 error 2005 later.

Reply:
ok